MayankPandey14.Json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Please add this json file in Data folder @MAYANKpandey14 
This might help: https://linkfree.eddiehub.io/docs/quickstart

I think you have merged all your previous commits in this branch, You should close this pr and create a new one with new branch and there you should create your Linktree data file. Thank you

Reply:
Ok @sidharthviz 
This is was due to my another pr i submitted earlier this week .